tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -29,10 +35,14 @@
 import net.sourceforge.argparse4j.inf.ArgumentParser;
 import net.sourceforge.argparse4j.inf.ArgumentParserException;
 import net.sourceforge.argparse4j.inf.Namespace;
+import org.apache.kafka.common.Metric;
+import org.apache.kafka.common.MetricName;
 
 public class ProducerPerformance {
+    private static final AtomicInteger id = new AtomicInteger(0);
```

Comment:
The reason it is put out side is in case there are multiple producer performance running in the same JVM. We don't want to have sensor collision in that case.